www/src/components/chat/chat-empty-state.tsx

```diff
@@ -9,32 +10,39 @@ export function ChatEmptyState({
 }: {
   onSuggestionClick: (suggestion: string) => void;
 }) {
+  const { isMobile } = useMobileStore();
+
   return (
     <div className="flex flex-col items-center justify-center h-full p-4">
       <div className="mb-8">
-      <Image
-            src="/op-logo.svg"
-            alt="Optimism logo"
-            width={100}
-            height={100}
-          />
+        <Image
+          src="/op-logo.svg"
+          alt="Optimism logo"
+          width={75}
+          height={75}
+          className="w-[75px] md:w-[100px]"
+        />
       </div>
-      <h2 className="text-xl md:text-2xl font-bold mb-4 text-center text-chat-secondary">
-        How can I help you today?
+      <h2 className="text-lg md:text-2xl font-bold mb-4 text-center text-chat-secondary">
+        {isMobile
+          ? "Ask me anything about Optimism Governance"
+          : "How can I help you today?"}
```

Comment:
why change this when on mobile?

Reply:
I was following [Figma](https://www.figma.com/design/TuQQU1u6ZnwjnlCDYbSCBS/OP-AI-Tools-UI?node-id=1-5&t=Aitfe2M0yYSb65jH-4), and thought it was intentional because,  the `how can I help you`, matches with the suggestions idea, and without the suggestion, maybe the other text make's more sense

Reply:
I've keep the "How can I help you today" for both mobile and desktop